### PR TITLE
Posts show page stylized

### PR DESCRIPTION
**Pull request by akamran2001**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -12,72 +10,73 @@
...
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 20_221_108_200_114) do
-  create_table 'comments', force: :cascade do |t|
-    t.string 'text_body'
-    t.datetime 'created_at', null: false
-    t.datetime 'updated_at', null: false
-    t.integer 'post_id'
-    t.index ['post_id'], name: 'index_comments_on_post_id'
+ActiveRecord::Schema[7.0].define(version: 2022_11_08_200114) do
```
Comment (jpaints3) [inline db/schema.rb:13]: Is there a benefit to this outside of stylistic changes?

**Review by jpaints3 — APPROVED** [review]
Looks good to me. I am worried about changing the schema version, but it might just be a style change.

File: app/views/posts/show.html.erb [hunk 2]
```diff
@@ -1,13 +1,54 @@
...
-
- <h2><%= link_to "Comment", new_course_post_comment_path(@course_id, @post_id) %></h2>
+<!--<div>-->
+<!--#TODO put some info about the question/post up here with comments below-->
+<!--<h1> Comments for this question </h1>-->
+
+
+<!-- <h2><%#= link_to "Comment", new_course_post_comment_path(@course_id, @post_id) %></h2>-->
+<!--</div>-->
+
```
[the remark this answers is not in the document]
Comment (akamran2001) [reply app/views/posts/show.html.erb:8]: I moved the header partial into the application.html.erb

[the remark this answers is not in the document]
Comment (jpaints3) [reply app/views/posts/show.html.erb:8]: Oh, that's smart, ok sounds good. 👍🏿

**Pull request merged**